zowex/src/main.rs

```diff
@@ -279,30 +290,48 @@ fn run_daemon_command(args: &mut Vec<String>) -> io::Result<()> {
         _resp.append(&mut stdin);
     }
 
-    // form our host, port, and connection strings
-    let daemon_host = "127.0.0.1".to_owned();
-    let port_string = get_port_string();
+    let mut tries = 0;
+    loop {
+        let socket_string = get_socket_string();
+        let mut stream = establish_connection(socket_string)?;
+        match talk(&_resp, &mut stream) {
+            Ok(result) => { return Ok(result); },
+            Err(ref e) if e.kind() == io::ErrorKind::ConnectionReset => {
+                if tries > THREE_MIN_OF_RETRIES {
+                    println!("Terminating after {} connection retries.", THREE_MIN_OF_RETRIES);
+                    std::process::exit(EXIT_CODE_TIMEOUT_CONNECT_TO_RUNNING_DAEMON);
+                }
 
-    let mut stream = establish_connection(daemon_host, port_string)?;
-    talk(&_resp, &mut stream)
+                println!("{} ({} of {})", "The Zowe daemon is in use, retrying", tries, THREE_MIN_OF_RETRIES);
```

Comment:
Clippy (the Rust linter) doesn't like this line: `literal with an empty format string`

Reply:
Done